src/showcase/ui-group/ButtonCards.tsx

```diff
@@ -9,6 +9,7 @@ import Link from 'next/link'
 import GradientFillButton from '@/showcase/components/button/GradientFillButton'
 import BorderGlowButton from '@/showcase/components/button/BorderGlowButton'
 import ShineButton from '@/showcase/components/button/ShineButton'
+import TextRevealButton from '../components/button/TextRevealButton'
```

Comment:
use `@/showcase/components/button/TextRevealButton`

Reply:
use client also not need i guess ??